products/statement-generator/src/contexts/FormStateProps.ts

```diff
@@ -166,57 +166,53 @@ export interface IStatementState {
 
 export const sampleStepState = {
   introduction: {
-    fullName: 'Daniel Xiao',
-    age: '29',
-    isVeteran: 'Yes',
+    fullName: process.env.REACT_APP_FULL_NAME,
+    age: process.env.REACT_APP_AGE,
+    isVeteran: process.env.REACT_APP_IS_VETERAN,
   },
   involvement: {
```

Comment:
It looks like this sample state hasn't been updated in a sec. I noticed that the involvement section is missing the `isSomethingElseChecked` property and has an extra `isNoneChecked` property. Can you also add the `somethingElseState` section?

Can you update it to match the properties in the `defaultStepState` variable on like 21?

Reply:
Copy. This will be in by tomorrow morning! 

See you at the meeting